jazzy.repos

```diff
@@ -17,6 +17,11 @@ repositories:
     # Until: https://github.com/novatel/novatel_oem7_driver/pull/89
     url: https://github.com/nobleo/novatel_oem7_driver.git
     version: fix/missing-git-dep
+  off_highway_sensor_drivers:
+    type: git
+    # Until: https://github.com/nobleo/off_highway_sensor_drivers
+    url: https://github.com/nobleo/off_highway_sensor_drivers.git
```

Comment:
Why our fork? It doesn't contain improvements on the upstream.

Reply:
I just copied from iron.yaml. So you can ask that question to the person who put it there :)